manifests/4.7/elasticsearch-operator.v4.7.0.clusterserviceversion.yaml

```diff
@@ -40,6 +40,7 @@ metadata:
             "spec": {
               "managementState": "Managed"
               "nodeSpec": {
+                "image": "quay.io/openshift/origin-logging-elasticsearch6:latest",
```

Comment:
The image is handled by the operator and does not use the image in the nodeSpec anymore:
https://github.com/openshift/elasticsearch-operator/blob/master/cmd/manager/main.go#L129

Reply:
Shall I remove the image in this PR? Thanks.

Reply:
Yes, please.

Reply:
Fixed. Thanks.